modules/core/src/main/java/org/apache/ignite/internal/cluster/IgniteClusterImpl.java

```diff
@@ -315,15 +317,27 @@ public IgniteClusterImpl(GridKernalContext ctx) {
 
         try {
             ctx.state().changeGlobalState(active, baselineNodes(), false).get();
+
+            recordEvent(active ? EventType.EVT_CLUSTER_ACTIVATED : EventType.EVT_CLUSTER_DEACTIVATED);
         }
         catch (IgniteCheckedException e) {
+            recordEvent(EventType.EVT_CLUSTER_ACTIVATION_FAILED);
```

Comment:
I'm not sure that we need this one. I can't find any other unhandled failure events.